### ISSUE TITLE
How to specify API key in a public project?

### ISSUE THREAD

**Issue by Olli-palkkaus** [crowdin/vscode-crowdin#30]
**Is your feature request related to a problem? Please describe.**
I have a public project in GitLab. If I configure the api_token as described in the instructions the API token would get to the repository which would mean that anyone could get access to translations. I am new to Crowdlin, so I do not know exactly, but I assume the API key would have access to quite a large set of features so this is probably a big security risk.

I cannot put the crowdin.yml in gitignore because then the integration through GitLab would not work.

**Describe the solution you'd like**
Ability to specify the API key in VSCode settings so that they can be machine specific, not in the project files that go to GIT. Can I use "Environment variables" here somehow? I am not aware of any way that I could specify them in VSCode...

**Describe alternatives you've considered**
Perhaps a separate file for project id and API key. This could then be put to GitIgnore.

**Comment by andrii-bodnar** [comment]
Hi @Olli-palkkaus !
There is a possibility to load the API Credentials from an **environment variables**, for example:

```
"project_id_env": "CROWDIN_PROJECT_ID"
"api_token_env": "CROWDIN_PERSONAL_TOKEN"
```

[API Credentials from Environment Variables](https://support.crowdin.com/configuration-file/#api-credentials-from-environment-variables)
[Configuration](https://github.com/crowdin/vscode-crowdin#configuration)

**Comment by Olli-palkkaus** [comment]
Sorry, in the evening I read and tried this, but I did not realize it was the **operating system** environment variable. They are used so rarely these days, so I was searching some kind of environment variable in Visual Studio code. And adding Windows env variable seems to require reboot before VSCode sees it (not 100% sure, didn't retest the behavior after I got it working).

But looking at it now with fresh eyes I realize that you probably did this config file first and and foremost to your CLI and VSCode extension is just reusing it.

So yes, I realize the idea now and got it working
